scripts/testnet-deployments/node.Dockerfile

```diff
@@ -1,25 +1,15 @@
-# syntax=docker/dockerfile:experimental
-
 ARG universal_image_name
 
-# Fetch genesis-data.
-FROM alpine/git:latest as genesis-data
-ARG genesis_ref
-ARG genesis_path
-RUN mkdir -p -m 0600 ~/.ssh && ssh-keyscan gitlab.com >> ~/.ssh/known_hosts
-RUN --mount=type=ssh git clone --depth 1 --branch "${genesis_ref}" git@gitlab.com:Concordium/genesis-data.git
-RUN mv "genesis-data/${genesis_path}" /data
-
```

Comment:
It's unclear to me what the purpose of this image is. It has an entrypoint `start.sh`, but it doesn't have any documentation on how to run it, what to set. Without genesis data it must assume that this data is mounted in some way in some place inside the image. This needs to be documented.

Reply:
I think this comment is outdated. The current image uses the binary directly for its entrypoint.